internal/database/connect_test.go

```diff
@@ -3,15 +3,25 @@ package database
 import (
 	"testing"
 
-	"github.com/source-academy/stories-backend/internal/config"
+	"github.com/source-academy/stories-backend/internal/testutils"
 
 	"github.com/stretchr/testify/assert"
 )
 
+const (
+	test_env_path  = "../../.env.testing"
+	migration_path = "../../migrations"
```

Comment:
This introduces some potential (albeit minor) maintainability issues as every time the folder structure changes, we might need to check that this works.

What was the rationale to remove the temporary env file creation logic and replace it with this?

Reply:
will be resolved in #81